src/main/java/com/github/manolo8/darkbot/core/manager/HeroManager.java

```diff
@@ -140,18 +141,35 @@ public boolean setMode(Config.ShipConfig config) {
         return setMode(config.CONFIG, config.FORMATION);
     }
 
+    private boolean formationCheckBySlot = true;
```

Comment:
put this above alongside the rest of the state for hero manager

also rename it to something that makes more sense, i'm not sure if this is `shouldCheckFormationBySlot` or `didCheckBySlot` or what